### PR TITLE
Add grant winner series blog post, from Alowina

### PR DESCRIPTION
**Pull request by WrathfulSpatula**

### CODE REVIEW

**Review by nathanshammah — CHANGES_REQUESTED** [review]
Thanks @WrathfulSpatula.
- Can you please include the md file? It is easier to review and it's good that we keep a copy in the markdown folder too, as in the future we may use that as a main reference in a new website build. I'll review the md file then. 
- Please make the title more explicit, like other ones: Even something like "Inaugurating the Grant Winner Video Series"
- @Misty-W please review and edit the blog post: It should include your perspective!

**Comment by natestemen** [comment]
> Can you please include the md file? It is easier to review and it's good that we keep a copy in the markdown folder too, as in the future we may use that as a main reference in a new website build.

Just FYI [this PR](https://github.com/unitaryfund/unitary-fund/pull/242) did not have an associated markdown file either. Not sure we'd really want to go back and change it, but thought I'd surface it in case this is a problem.

**Review by natestemen — DISMISSED** [review]
LGTM! 🚢

**Comment by nathanshammah** [comment]
@natestemen no need to do extra work now, but whenever possible, when we catch it, I'd aim to include the .md file for consistency. It also simplifies review of the text (which I haven't done yet) as it does not force one to fork this and host it locally. It takes one line with `pandoc` to generate it. 

For @Misty-W, integrating my previous comments above:

- Please include the youtube-nocookie link as embedded video, e.g. as done for the [Metriq blog](https://unitary.fund/posts/metriq_release.html) post, so that the video window previews in the page. 

- Update date in the blog index and post page (it may be needed to rebase it to include the recent additions from the UF interns).

File: posts/grant_winner_series.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+                            <right>
+                                <h2 class="leading-arrow">Inaugurating the Grant Winner Video Series</h2>
+                                <h3></h3>
+                            </right>
+                            2022-08-18
+                            <br> Alowina Yap
+
+
+                            <div>
+                                <p>We are very excited to share our first video for Unitary Fund’s Grant Winner series! The series will feature some of our grant winners, their projects, their experience after winning the grant, and what they’re working on now.</p>
```
Comment (Misty-W) [inline posts/grant_winner_series.html:80]:
```suggestion
                                <p>We are very excited to share our first episodes of Unitary Fund’s Grant Winner video series! Watch to learn more about our featured grant winners, their projects, their experience after winning the grant, and what they’re working on now.</p>
```

File: posts/grant_winner_series.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                                <p>Hosted by our very own, Misty Wahl– our first video features Pedro Rivero, author of QRAND and our second video is with Lia Yeh, Samanvay Sharma, and Alberto Maldonado with PIQUE.</p>
```
Comment (Misty-W) [inline posts/grant_winner_series.html:81]:
```suggestion
                                <p>Hosted by UF Technical Staff Member Misty Wahl– our first video features Pedro Rivero, author of QRAND and our second video is with Lia Yeh, Samanvay Sharma, and Alberto Maldonado with PIQUE.</p>
```

**Review by Misty-W — CHANGES_REQUESTED** [review]

File: posts/_markdown/grant_winner_series.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Inaugurating the Grant Winner Video Series
+date: 18 February 2022
```
Comment (Misty-W) [inline posts/_markdown/grant_winner_series.md:3]:
```suggestion
date: 26 August 2022
```

**Review by Misty-W — CHANGES_REQUESTED** [review]

File: posts/grant_winner_series.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+                                <p><a href="https://youtu.be/LSOCHWSPvUc">About QRAND</a>: QRAND introduces a versatile interface layer between NumPy and several quantum computing platforms (qiskit, cirq, qsharp...), along with some useful functionality that enables the production of quantum random numbers (QRN) according to different quantum protocols, and for a wide variety of probability distributions.</p>
```
Comment (Misty-W) [inline posts/grant_winner_series.html:82]:
```suggestion
              <iframe
                width="560"
                height="315"
                src="https://www.youtube-nocookie.com/embed/LSOCHWSPvUc""
                title="YouTube video player"
                style="border: none"
                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
                allowfullscreen
              ></iframe>
              <p>About QRAND</a>: QRAND introduces a versatile interface layer between NumPy and several quantum computing platforms (qiskit, cirq, qsharp...), along with some useful functionality that enables the production of quantum random numbers (QRN) according to different quantum protocols, and for a wide variety of probability distributions.</p>
```

**Review by Misty-W — CHANGES_REQUESTED** [review]

**Review by Misty-W — APPROVED** [review]

**Review by nathanshammah — APPROVED** [review]
LGMT, thanks Misty!

**Pull request merged**